### ISSUE TITLE
There was a gradient explosion when I'm training data. Why is the learning rate rising all the time

### ISSUE THREAD

**Issue by leofionn** [hunglc007/tensorflow-yolov4-tflite#138]
I use this code to train my own data It often occurs that the loss function is NAN.
![9921235a9cc65965f17312b7e156728](https://user-images.githubusercontent.com/33302972/87003996-43f9a480-c1ef-11ea-9940-1b1ec9725cf3.jpg)
These loss functions are not convergent,the learning rate rising all the time.
Someone can tell me what's going on here? I'm a student,and my technique is not very good.
thanks!

**Comment by leofionn** [comment]
I set a very small learning rate:
__C.TRAIN.LR_INIT  = 1e-5
__C.TRAIN.LR_END = 1e-8
Or will there be a gradient explosion
![image](https://user-images.githubusercontent.com/33302972/87004622-4c061400-c1f0-11ea-91d5-1a6f076712b8.png)

**Comment by SuperCrystal** [comment]
Are you still in the warm-up epochs? In the very first stages the lr rate will be rising from a small value to your LR_INIT

**Comment by leofionn** [comment]
> Are you still in the warm-up epochs? In the very first stages the lr rate will be rising from a small value to your LR_INIT

NO,  I'm sure it's not in the warm up epoch phase..I also want to know Why is the learning rate rising.

**Comment by hunglc007** [comment]
I updated mish activation to fix nan loss issue.

**Comment by leofionn** [comment]
> I updated mish activation to fix nan loss issue.

When I use JPG image, it is OK, but if I use TIFF format, Nan will appear.
I will use the new activation. Thank you for your help
